lib/WUI/wui.c

```diff
@@ -72,13 +72,14 @@ static void update_eth_changes(void) {
 }
 
 void StartWebServerTask(void const *argument) {
+    wui_eth_config.var_mask = ETHVAR_EEPROM_CONFIG;
+    load_eth_params(&wui_eth_config);
```

Comment:
Looking at main.cpp and order of initialization (esp. EEPROM) it looks like the strange osDelay(1000) can be finally removed and loading data from EEPROM is safe here.